pages/education.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Instructor Education</title>
+  <link rel="stylesheet" href="../stylesheets/style.css">
+  <link rel="stylesheet" href="../stylesheets/education.css">
+</head>
+<body>
+    <header>
```

Comment:
The `header` tag is more so used for introductory content. Here we are wrapping the most, if not all, the page content in this tag. We could replace this tag with something like `main` instead.